From ed87c0942c9967a4da9052717b81d48c43d1b1aa Mon Sep 17 00:00:00 2001
From: Alicja Michalska <ahplka19@gmail.com>
Date: Sat, 25 Nov 2023 04:16:44 +0100
Subject: [PATCH 11/11] arm64: dts: mediatek: mt8183-kukui: Change displayed
 names

Make boardnames appear nicer to end-user, representing the actual device
name as sold by manufacturer.

Signed-off-by: Alicja Michalska <ahplka19@gmail.com>
---
 .../boot/dts/mediatek/mt8183-kukui-jacuzzi-fennel14-sku2.dts    | 2 +-
 arch/arm64/boot/dts/mediatek/mt8183-kukui-jacuzzi-fennel14.dts  | 2 +-
 arch/arm64/boot/dts/mediatek/mt8183-kukui-krane-sku0.dts        | 2 +-
 arch/arm64/boot/dts/mediatek/mt8183-kukui-krane-sku176.dts      | 2 +-
 4 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/arch/arm64/boot/dts/mediatek/mt8183-kukui-jacuzzi-fennel14-sku2.dts b/arch/arm64/boot/dts/mediatek/mt8183-kukui-jacuzzi-fennel14-sku2.dts
index 3fc5a6181d7e..0237a3426f4a 100644
--- a/arch/arm64/boot/dts/mediatek/mt8183-kukui-jacuzzi-fennel14-sku2.dts
+++ b/arch/arm64/boot/dts/mediatek/mt8183-kukui-jacuzzi-fennel14-sku2.dts
@@ -8,7 +8,7 @@
 #include "mt8183-kukui-audio-ts3a227e-rt1015p.dtsi"
 
 / {
-	model = "Google fennel14 sku2 board";
+	model = "Lenovo IdeaPad 3 Chromebook (SKU2)";
 	compatible = "google,fennel-sku2", "google,fennel", "mediatek,mt8183";
 };
 
diff --git a/arch/arm64/boot/dts/mediatek/mt8183-kukui-jacuzzi-fennel14.dts b/arch/arm64/boot/dts/mediatek/mt8183-kukui-jacuzzi-fennel14.dts
index 23ad0b91e977..12b04ea64e13 100644
--- a/arch/arm64/boot/dts/mediatek/mt8183-kukui-jacuzzi-fennel14.dts
+++ b/arch/arm64/boot/dts/mediatek/mt8183-kukui-jacuzzi-fennel14.dts
@@ -8,7 +8,7 @@
 #include "mt8183-kukui-audio-da7219-rt1015p.dtsi"
 
 / {
-	model = "Google fennel14 sku0 board";
+	model = "Lenovo IdeaPad 3 Chromebook";
 	compatible = "google,fennel-sku0", "google,fennel", "mediatek,mt8183";
 };
 
diff --git a/arch/arm64/boot/dts/mediatek/mt8183-kukui-krane-sku0.dts b/arch/arm64/boot/dts/mediatek/mt8183-kukui-krane-sku0.dts
index 4ac75806fa94..e1130cb252a5 100644
--- a/arch/arm64/boot/dts/mediatek/mt8183-kukui-krane-sku0.dts
+++ b/arch/arm64/boot/dts/mediatek/mt8183-kukui-krane-sku0.dts
@@ -13,7 +13,7 @@
 #include "mt8183-kukui-krane.dtsi"
 
 / {
-	model = "MediaTek krane sku0 board";
+	model = "Lenovo Chromebook Duet (AUO)";
 	chassis-type = "tablet";
 	compatible = "google,krane-sku0", "google,krane", "mediatek,mt8183";
 };
diff --git a/arch/arm64/boot/dts/mediatek/mt8183-kukui-krane-sku176.dts b/arch/arm64/boot/dts/mediatek/mt8183-kukui-krane-sku176.dts
index 095279e55d50..35a0ad99f3ac 100644
--- a/arch/arm64/boot/dts/mediatek/mt8183-kukui-krane-sku176.dts
+++ b/arch/arm64/boot/dts/mediatek/mt8183-kukui-krane-sku176.dts
@@ -13,7 +13,7 @@
 #include "mt8183-kukui-krane.dtsi"
 
 / {
-	model = "MediaTek krane sku176 board";
+	model = "Lenovo Chromebook Duet (BOE)";
 	chassis-type = "tablet";
 	compatible = "google,krane-sku176", "google,krane", "mediatek,mt8183";
 };
-- 
2.41.0

